### PR TITLE
Bugfix - Fixing Attribute Error 

### PR DESCRIPTION
**Pull request by THOR300**
# Description

Traceback of the error experienced in the backend code: 

          AttributeError: 'Language' object has no attribute 'code'
          
This error comes from the following line:

https://github.com/climatepolicyradar/navigator-backend/blob/main/app/db/crud/document.py#L105 

Please include:
- a summary of the changes
- links to any related issue/ticket
- any additional relevant motivation and context
- details of any dependency updates that are required for this change

## Type of change

Please select the option(s) below that are most relevant:

- [X] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

**Review by diversemix — CHANGES_REQUESTED** [review]
The modification to the if statement is not needed see:
```
>>> language = None
>>> print("Present" if language else "None")
None
>>> language = "chips"
>>> print("Present" if language else "None")
Present
>>> 
```

File: app/db/crud/document.py [hunk 1]
```diff
@@ -102,7 +102,9 @@ def _get_language_for_phys_doc(db: Session, physical_document_id: str) -> str:
         .filter(Language.id == PhysicalDocumentLanguage.language_id)
     ).one_or_none()
 
-    return cast(str, language.code) if language else ""
+    # I do not fully understand why the strip() below is required, but without it
+    # we see trailing spaces on the language code.
```
Comment (joel-wright) [inline app/db/crud/document.py:106]: Can anyone explain this?!

Comment (THOR300) [reply app/db/crud/document.py:106]: Language code is length 3? 
So if it's en it's entered as 'en ' ?

Comment (joel-wright) [reply app/db/crud/document.py:106]: good spot! should probably use the real value of "eng" in the test

**Comment by THOR300** [comment]
> The modification to the if statement is not needed see:
> 
> ```
> >>> language = None
> >>> print("Present" if language else "None")
> None
> >>> language = "chips"
> >>> print("Present" if language else "None")
> Present
> >>> 
> ```

The error must just have been from code actually being language_code then if one_or_none truly returns NoneType.

**Pull request merged**